src/components/dashboard/ClustersWithIssues/ClustersWithIssues.module.scss

```
@@ -0,0 +1,41 @@
+  padding: 0;
+  cursor: pointer;
+  font: inherit;
+  color: var(--pf-t--global--text--color--link--default);
+  text-decoration: none;
+
+  &:hover {
+    text-decoration: underline;
+  }
+
+  &:focus-visible {
+    outline: 2px solid currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)